packages/client/components/TopBarSearch.tsx

```diff
@@ -43,7 +43,7 @@ const Wrapper = styled('div')<{location: any}>(({location}) => ({
   display: 'flex',
   flex: 1,
   height: 40,
-  margin: '8px 16px',
+  margin: 8,
```

Comment:
@BartoszJarocki curious why are you not using `px` units here?

Reply:
it's automatically converted to px by styled components. specifying unit is needed when you want to use a shorthand version, like it was before